docs/README.md

```diff
@@ -24,6 +24,16 @@ Theme is copied from Daml Docs theme.
 
 `Node` and `yarn` must be installed on your machine.
 
+### Sass
+
+The `Node` version of `sass` must be installed: `gem install sass`
+
+### Grunt
+
+If grunt is not already installed: `sudo npm install -g grunt-cli`
```

Comment:
Do you really need `sudo` here?

Reply:
I tried without and got the following:
```
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
```

Reply:
the `-g` install the package globally, hence the need for `sudo`. Probably works fine without `sudo` and without `-g`